tensorflow/core/api_def/base_api/api_def_Lgamma.pbtxt

```diff
@@ -1,4 +1,13 @@
 op {
   graph_op_name: "Lgamma"
   summary: "Computes the log of the absolute value of `Gamma(x)` element-wise."
+  description: <<END
+  This function computes log((input - 1)!) for every element in the tensor.
+  lgamma(5) = log((5-1)!) = log(4!) = log(24) = 3.1780539
```

Comment:
How does this work for non-integers? It would be worth saying that gamma is the continuous equivalent of !, not really !, otherwise people will get confused seeing the example below.

Reply:
I have modified it as positive numbers. I did not add formula for non-integer negative numbers.
Reference: https://mc-stan.org/docs/2_20/functions-reference/gamma-appendix.html

Do you think this is ok? @martinwicke